apps/session-manager/api/fims.yaml

```diff
@@ -67,6 +135,26 @@ definitions:
           - auth_time
           - acr
       - $ref: "#/definitions/FederatedUser"
+  FIMSPlusUser:
+    type: object
+    properties:
+      user:
```

Comment:
accordingly with the scope naming in OIDC this prop could be named `profile` instead of `user`

Reply:
I maintained consistency with the existing FIMS endpoint, which path is `/fims/api/v1/user`

Reply:
I know, I wanted to maintain consistency, but if you prefer this way it's fine for me

Reply:
Done in [ad1e3fc](https://github.com/pagopa/io-auth-n-identity-domain/pull/27/commits/ad1e3fc85b651d654f6d996a65a45ea98af9c42e)